aws/lambda-and-api-gateway/lambda-api-gateway/lambda-api-gateway.md

```diff
@@ -105,6 +105,47 @@ callback(null, response);
 
 Depending on whether or not this is a successful case, you can put in other status codes. [Refer to this page](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes) to find a status code that is appropriate for your needs.
 
+## Cloudwatch Logs
+
+As you get further into development, there may be times where you can no longer check the Lambda console or the API gateway testing environment to view your results. This might be because you are calling a backend function through the front end to see if it is working as intended with the added layer of complexity. In these situations, you will need a way to check the lambda function's outputs, or more importantly, their error messages.
```

Comment:
Call "front end" "UI", and edit the last sentence to "In these situations, you will need a way to check the Lambda function's console logs, or more importantly, their error messages."